test/jasmine/tests/hover_label_test.js

```diff
@@ -574,3 +574,34 @@ describe('hover info on stacked subplots', function() {
         });
     });
 });
+
+
+describe('hover info on overlaid subplots', function() {
+    'use strict';
+
+    afterEach(destroyGraphDiv);
+
+    it('should respond to hover', function(done) {
```

Comment:
might as well test this [non-trivial](https://github.com/plotly/plotly.js/pull/575/commits/858a3388a316544ee298ef86d65d70c83a160426#diff-c3362661b0796b176be95487fa368e39R323) logic in `Fx.hover`.